frontend/src/metabase/dashboard/hoc/DashboardControls.jsx

```diff
@@ -174,6 +174,7 @@ export default ComposedComponent =>
             { preserveParameters: true },
           );
           this.props.fetchDashboardCardData({
+            isRefreshing: true,
```

Comment:
So others also use this. [Ref from Apple doc](https://developer.apple.com/documentation/uikit/uirefreshcontrol/1624844-isrefreshing).

As a side note, naming this option is very refreshing.